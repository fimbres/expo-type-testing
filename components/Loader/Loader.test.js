import React from 'react';
import renderer from 'react-test-renderer';

import { Loader } from '.';

let tree;

describe('<Loader />', () => {
    beforeEach(() => {
        tree = renderer
            .create(
                <Loader />
            ).toJSON();
    });

    it('have 1 children', () => {
        expect(tree.children.length).toBe(1);
        expect(tree).toMatchSnapshot();
    });

    it('render correctly', () => {
        expect(tree).toBeDefined();
        expect(tree).toMatchSnapshot();
    });
});
