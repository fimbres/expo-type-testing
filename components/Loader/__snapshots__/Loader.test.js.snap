// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader /> have 1 children 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#020202CC",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "zIndex": 5,
    }
  }
  testID="loader"
>
  <ActivityIndicator
    color="#CFD6FF"
    size="large"
  />
</View>
`;

exports[`<Loader /> render correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#020202CC",
      "bottom": 0,
      "justifyContent": "center",
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 0,
      "zIndex": 5,
    }
  }
  testID="loader"
>
  <ActivityIndicator
    color="#CFD6FF"
    size="large"
  />
</View>
`;
